pkg/plugins/discovery/k8s/controllers/pod_converter_test.go

```diff
@@ -329,6 +337,445 @@ var _ = Describe("PodToDataplane(..)", func() {
                     version: "0.1"
 `,
 		}),
+		Entry("pod with direct access to all services", testCase{
```

Comment:
Yaml configs in entries are quite big, I believe it would be much simpler to read tests if configs were placed in `testdata`

Reply:
Took me a while but it's done